### PR TITLE
upgrade docker infrastructure to get the latest test-containers

### PR DESCRIPTION
**Pull request by pbruski**

### CODE REVIEW

File: build.gradle.kts [hunk 1]
```diff
@@ -55,7 +55,7 @@ dependencies {
     }.forEach { implementation(it) }
     testCompile("org.assertj:assertj-core:3.11.0")
     testCompile("com.atlassian.performance.tools:io:[1.0.0,2.0.0)")
-    testCompile("com.atlassian.performance.tools:docker-infrastructure:0.3.1")
+    testCompile("com.atlassian.performance.tools:docker-infrastructure:0.3.2")
```
Comment (wyrzyk) [inline build.gradle.kts]: Why not a range?

Comment (dagguh) [reply build.gradle.kts]: It's not stable 1.0.0+ yet

File: CHANGELOG.md [hunk 2]
```diff
@@ -25,6 +25,8 @@ Dropping a requirement of a major version of a dependency is a new contract.
 
 ## [Unreleased]
 [Unreleased]: https://github.com/atlassian/jira-actions/compare/release-3.10.2...master
+### Changed
```
Comment (dagguh) [inline CHANGELOG.md]: First of all, it's not changed, but fixed.
Second of all, test dependencies are not in the POM, so they're not API.
Let's remove this.

Comment (pbruski) [reply CHANGELOG.md]: https://github.com/atlassian/jira-actions/pull/56

**Pull request merged**